app/src/main/java/com/kkolontay/theworld/ui/screens/WorldNavigation.kt

```diff
@@ -27,70 +30,60 @@ enum class AppScreens() {
 
 @Composable
 fun WorldNavigation(
-   // viewModel: OrderViewModel = viewModel(),
+    viewModel: CountryViewModel = androidx.lifecycle.viewmodel.compose.viewModel(),
```

Comment:
Great job here! I think you should be able to statically import this `viewModel()` method as well to reduce the namespace here:

```kotlin
import androidx.lifecycle.viewmodel.compose.viewModel()
...

@Composable
fun WorldNavigation(
    viewModel: CountryViewModel = viewModel(),
    ...
```